frontend/src/v4/modules/tree/tree.selectors.ts

```diff
@@ -59,6 +58,11 @@ export const selectTreeNodesList = createSelector(
 	(treeProcessingData) => treeProcessingData.nodesList || []
 );
 
+export const selectSortedTreeNodesList = createSelector(
+	selectTreeNodesList,
+	(nodesList) => orderBy(nodesList, 'name')
```

Comment:
I think there needs to be a toLowerCase() somewhere here because currently lowercase nodes are always ordered after uppercase nodes

Reply:
And perhaps in the transforming,ts file too